proposals/0013-expanded-comparison-sampling.md

```diff
@@ -0,0 +1,432 @@
+# Expanded Comparison Sampling
+
+* Proposal: [0013](0013-expanded-comparison-sampling.md)
+* Author(s): [Greg Roth](https://github.com/pow2clk),
+             [Jesse Natalie](https://github.com/jenatali)
+* Sponsor: [Greg Roth](https://github.com/pow2clk)
+* Status: **Under Consideration**
+* Planned Version: Shader Model 6.8, validator version 1.8
```

Comment:
Is the validator version really relevant here? This feature is part of 6.8, the fact that the support lands in 1.8 validator or 1.8.xxx DXC seems like an implementation detail.

Reply:
We've occasionally toyed with the idea of divorcing the validator version from the shader model version. I did it mainly because Tex did in #112, but I'm not going to insist on it.

Reply:
Oh, I included it in that PR because availability of this new info in PSV0 for the runtime is only dependent on validator version.  In fact, I should probably remove "SM 6.8" because it's not dependent on shader model.